plenum/server/replica.py

```diff
@@ -946,15 +951,16 @@ def report_suspicious(reason):
             pp_view_no = pre_prepare.viewNo
             pp_seq_no = pre_prepare.ppSeqNo
             last_pp_view_no, last_pp_seq_no = self.__last_pp_3pc
-            if pp_view_no >= last_pp_view_no:
+            if pp_view_no >= last_pp_view_no and (
+                    self.isMaster or not self.last_ordered_3pc[1] != 0):
```

Comment:
Should it be `self.last_ordered_3pc[1] == 0`?

Reply:
fixed it